Discovery/Web-Content/combined_directories.txt

```diff
@@ -237176,6 +237176,8 @@ granted
 group_inlinemod
 gv_redeem
 gv_send
+h2-console
```

Comment:
You should not contribute to the `combined_directories.txt` wordlist directly; direct changes will be overwritten by the github action. Instead, please add your contributions to one of the lists that composes `combined_directories.txt`:

- apache.txt
- combined_words.txt
- directory-list-1.0.txt
- directory-list-2.3-big.txt
- directory-list-2.3-medium.txt
- directory-list-2.3-small.txt
- raft-large-directories-lowercase.txt
- raft-large-directories.txt
- raft-medium-directories-lowercase.txt
- raft-medium-directories.txt
- raft-small-directories-lowercase.txt
- raft-small-directories.txt

pick wichever you think is appropiate